requirements/base.in

```diff
@@ -1,5 +1,6 @@
 -c constraints.in
 unstructured[all-docs]>=0.8.1
+unstructured-client==0.25.5
```

Comment:
Overall LGTM. Why did this need to be pinned though?

Reply:
Investigating a failing test
- https://github.com/Unstructured-IO/unstructured-api/actions/runs/10793859754/job/29947819866?pr=458